templates/user/profile.html

```diff
@@ -73,6 +73,12 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
     )
     apiFeedback(call, 'Your username has been updated.')
   })
+  
+  const publishField = document.querySelector('[name=published]')
+  publishField.addEventListener('change', event => {
+    apiFeedback(call, 'Your article is publish preference has been updated.')
```

Comment:
It could be nice to have a message that mentions the state:
"You article has been published." or "You article has been unpublished."